cmd/lakefs/cmd/run.go

```diff
@@ -244,6 +244,10 @@ var runCmd = &cobra.Command{
 			}
 		}
 
+		if !emailer.IsEmailerConfigured() {
+			logger.Warn(fmt.Sprintf("Emailer has not been properly configured, check the values in the email config"))
+		}
```

Comment:
- This will now log a warning to all users that did not configure email (all our current users)
- This function doesn't only return if the email is configured but also validates it was configured correctly, In case it wasn't configured correctly we should provide information why (this is returned by `mail.ParseAddress` )
- If a user does configure invalid email parameters he will probably miss the warning

Reply:
Fixed. It will be fatal in the case the sender is misconfigured with the proper error.